deployment/ccip/changeset/home_chain.go

```diff
@@ -0,0 +1,77 @@
+package changeset
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/ccip-owner-contracts/pkg/proposal/timelock"
+
+	"github.com/smartcontractkit/chainlink/deployment"
+	ccipdeployment "github.com/smartcontractkit/chainlink/deployment/ccip"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/ccip/generated/rmn_home"
+	"github.com/smartcontractkit/chainlink/v2/core/gethwrappers/keystone/generated/capabilities_registry"
+)
+
+var _ deployment.ChangeSet = DeployHomeChain
+
+// DeployHomeChain is a separate changeset because it is a standalone deployment performed once in home chain for the entire CCIP deployment.
+func DeployHomeChain(env deployment.Environment, config interface{}) (deployment.ChangesetOutput, error) {
+	cfg, ok := config.(DeployHomeChainConfig)
+	if !ok {
+		return deployment.ChangesetOutput{}, deployment.ErrInvalidConfig
+	}
+	err := cfg.Validate()
+	if err != nil {
+		return deployment.ChangesetOutput{}, errors.Wrapf(deployment.ErrInvalidConfig, "%v", err)
+	}
+	ab := deployment.NewMemoryAddressBook()
```

Comment:
This is intentional. wanted to - keep env's Existing address unchanged, and then once the changeset is executed merge the address book with newly generated addresses. 
In this way we can know from each changeset which addresses got deployed.
Based on discussion with @connorwstein the format of every changeset should be -
```
func DoThingChangeset(e deployment.Environment, config interface {}) deployment.ChangesetOutput {
	// create new address book each time so that changeset output can denote 
	// which addresses are newly deployed
	newAddresses := deployment.NewAddressBook()
	// deployX should mutate newAddresses adding them.
	// it can use e.ExistingAddresses to load existing state
	// and do checks
	deployX(newAddresses, e)
	return Output{addresses: newAddresses}
} 
```